docs/demo-slides.md

```diff
@@ -34,7 +34,7 @@ source venv/bin/activate.fish
 Install CLI
 
 ```fish
-pip install git+https://github.com/instructlab/cli.git@stable
+pip install git+https://github.com/instructlab/instructlab.git@stable
```

Comment:
Now that instructlab is in PyPI, shouldn't we recommend `pip install instructlab` instead?

Reply:
Agreed.

Reply:
Also, we are deprecating the `stable` branch, so this reference also needs to be removed.

Reply:
Good point. Done.